package model;

public class Room {

    private String id;
    private int capacity;
    private String type;

    public Room(String id, int capacity, String type) {
        this.id = id;
        this.capacity = capacity;
        this.type = type;
    }

    public String getId() {
        return id;
    }

    public int getCapacity() {
        return capacity;
    }

    public String getType() {
        return type;
    }
}